src/static/index.css

```diff
@@ -0,0 +1,69 @@
+body, html {
+    height: 100%;
+    width: 100%;
+    background-color: rgb(34, 34, 34);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#tsparticles {
+    opacity: 0;
+    width: 0;
+    z-index: 0;
+    animation: fadeInBegin 2s ease forwards;
+}
+
+#tsparticles2 {
+    opacity: 0;
+    width: 0;
+    z-index: 0;
+    animation: fadeInBegin 3s ease-in forwards;
+}
+
+@keyframes fadeInBegin {
+    to { opacity: 1; }
+}
+
+#field {
+    width: 50vw;
+    padding: 2rem;
+    border: double 3px transparent;
+    border-radius: 6px;
+    background-image: linear-gradient(rgb(95, 95, 95), rgb(95, 95, 95)), 
+                      linear-gradient(to right, rgb(123, 149, 233), rgb(214, 98, 179));
+    background-origin: border-box;
+    background-clip: padding-box, border-box;
+    font-size: 24px;
+    outline: none;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

Comment:
declare Fon-family on top of your html instead of define it evey where.

Reply:
https://github.com/atyrode/vite/pull/12/commits/3c32e4953f7d3058d5631b405e926bd9a768a9a4